xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodec.h

```diff
@@ -75,6 +75,8 @@ struct VideoPicture
 
   unsigned int iWidth;
   unsigned int iHeight;
+  size_t iXOffset;
+  size_t iYOffset;
```

Comment:
There is probably no need to use `size_t` here, suggest use of `unsigned int` same as width and height for consistency.

Also `uint32_t GetXOffset()` is used in `CVideoBufferDRMPRIME`, so probably no need to keep the offset as 64-bit and truncate it.

Reply:
yes, they are originally unsigned int, i had just changed them to size_t only for testing.

And actually they are not initialized to 0, thats where the random uninitialized numbers are coming from, so it was an obvious bug from my end from the beginning.

Reply:
@Kwiboo also fixed in https://github.com/xbmc/xbmc/pull/24431/commits/1ecb17125a353bbd914c423f114f16c2ce889c01